pkg/component/component.go

```diff
@@ -166,7 +166,7 @@ func GetCurrent(client *occlient.Client, applicationName string, projectName str
 
 // PushLocal push local code to the cluster and trigger build there.
 // asFile indicates if it is a binary component or not
-func PushLocal(client *occlient.Client, componentName string, applicationName string, path string, asFile bool, out io.Writer) error {
+func PushLocal(client *occlient.Client, componentName string, applicationName string, path string, asFile bool, out io.Writer, files []string) error {
```

Comment:
So we still need `asFile` flag? We could just use `files` array for files and `path` for a directory.

Reply:
we will need that in case of binary, right ?

Reply:
removed dependency of `asFile` flag